test/k8s-integration/config/test-config.yaml

```diff
@@ -0,0 +1,21 @@
+ShortName: pdtest
```

Comment:
this file should not be checked in.  maybe we should add to .gitignore?

Reply:
@msau42 resolved :+1: